.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] You are not using correct Gitflow. In Gitlfow you make a development branch from the main/master branch and then you make all your feature branches from development branch and after making changes you merge that feature branch into your development branch 😊

Reply:
I followed the procedure that you are explaining here, first I created the master branch, then I created the dev branch from master, and the rest of the branches I created from the dev branch.

Reply:
I'm using correct Gitflow